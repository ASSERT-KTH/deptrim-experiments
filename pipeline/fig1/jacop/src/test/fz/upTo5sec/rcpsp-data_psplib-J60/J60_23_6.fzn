predicate jacop_cumulative(array [int] of var int: s,array [int] of var int: d,array [int] of var int: r,var int: b);
array [1..2] of int: X_INTRODUCED_62 = [1,-1];
array [1..31] of int: X_INTRODUCED_121 = [8,6,10,4,9,3,8,2,5,10,6,5,3,8,7,10,9,2,2,9,1,8,4,4,9,2,1,1,2,10,8];
array [1..31] of int: X_INTRODUCED_122 = [4,5,3,1,9,10,1,2,6,4,4,10,5,6,1,4,8,7,1,6,6,1,10,7,7,4,4,4,2,4,2];
array [1..29] of int: X_INTRODUCED_124 = [4,5,5,10,6,1,5,4,7,10,4,2,6,6,2,2,7,2,3,10,8,4,4,1,10,5,6,4,2];
array [1..29] of int: X_INTRODUCED_125 = [2,5,3,8,1,10,1,2,4,7,7,8,5,6,1,6,4,6,7,10,10,6,6,4,7,4,4,3,2];
array [1..30] of int: X_INTRODUCED_127 = [6,9,6,9,5,3,4,8,4,6,10,3,1,7,5,10,10,10,4,10,4,10,2,4,10,3,2,2,2,7];
array [1..30] of int: X_INTRODUCED_128 = [5,3,5,3,4,1,1,9,10,2,6,6,4,5,7,5,8,1,4,6,7,1,10,6,6,5,6,4,4,4];
array [1..31] of int: X_INTRODUCED_130 = [1,6,9,5,2,3,9,9,2,2,4,8,9,7,5,5,2,1,10,5,6,10,8,4,1,7,2,2,4,5,7];
array [1..31] of int: X_INTRODUCED_131 = [4,2,3,8,1,9,10,5,8,7,6,4,7,5,8,1,6,4,4,8,3,10,1,10,7,6,4,4,2,4,2];
var 0..311: X_INTRODUCED_0;
var 0..311: X_INTRODUCED_1;
var 0..311: X_INTRODUCED_2;
var 0..311: X_INTRODUCED_3;
var 0..311: X_INTRODUCED_4;
var 0..311: X_INTRODUCED_5;
var 0..311: X_INTRODUCED_6;
var 0..311: X_INTRODUCED_7;
var 0..311: X_INTRODUCED_8;
var 0..311: X_INTRODUCED_9;
var 0..311: X_INTRODUCED_10;
var 0..311: X_INTRODUCED_11;
var 0..311: X_INTRODUCED_12;
var 0..311: X_INTRODUCED_13;
var 0..311: X_INTRODUCED_14;
var 0..311: X_INTRODUCED_15;
var 0..311: X_INTRODUCED_16;
var 0..311: X_INTRODUCED_17;
var 0..311: X_INTRODUCED_18;
var 0..311: X_INTRODUCED_19;
var 0..311: X_INTRODUCED_20;
var 0..311: X_INTRODUCED_21;
var 0..311: X_INTRODUCED_22;
var 0..311: X_INTRODUCED_23;
var 0..311: X_INTRODUCED_24;
var 0..311: X_INTRODUCED_25;
var 0..311: X_INTRODUCED_26;
var 0..311: X_INTRODUCED_27;
var 0..311: X_INTRODUCED_28;
var 0..311: X_INTRODUCED_29;
var 0..311: X_INTRODUCED_30;
var 0..311: X_INTRODUCED_31;
var 0..311: X_INTRODUCED_32;
var 0..311: X_INTRODUCED_33;
var 0..311: X_INTRODUCED_34;
var 0..311: X_INTRODUCED_35;
var 0..311: X_INTRODUCED_36;
var 0..311: X_INTRODUCED_37;
var 0..311: X_INTRODUCED_38;
var 0..311: X_INTRODUCED_39;
var 0..311: X_INTRODUCED_40;
var 0..311: X_INTRODUCED_41;
var 0..311: X_INTRODUCED_42;
var 0..311: X_INTRODUCED_43;
var 0..311: X_INTRODUCED_44;
var 0..311: X_INTRODUCED_45;
var 0..311: X_INTRODUCED_46;
var 0..311: X_INTRODUCED_47;
var 0..311: X_INTRODUCED_48;
var 0..311: X_INTRODUCED_49;
var 0..311: X_INTRODUCED_50;
var 0..311: X_INTRODUCED_51;
var 0..311: X_INTRODUCED_52;
var 0..311: X_INTRODUCED_53;
var 0..311: X_INTRODUCED_54;
var 0..311: X_INTRODUCED_55;
var 0..311: X_INTRODUCED_56;
var 0..311: X_INTRODUCED_57;
var 0..311: X_INTRODUCED_58;
var 0..311: X_INTRODUCED_59;
var 0..312: objective:: output_var;
var bool: X_INTRODUCED_119 ::var_is_introduced :: is_defined_var;
var bool: X_INTRODUCED_120 ::var_is_introduced :: is_defined_var;
array [1..60] of var int: s:: output_array([1..60]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..31] of var int: X_INTRODUCED_123 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_8,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_22,X_INTRODUCED_24,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_48,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..29] of var int: X_INTRODUCED_126 ::var_is_introduced  = [X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_6,X_INTRODUCED_8,X_INTRODUCED_10,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_38,X_INTRODUCED_40,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_47,X_INTRODUCED_50,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_57];
array [1..30] of var int: X_INTRODUCED_129 ::var_is_introduced  = [X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_14,X_INTRODUCED_17,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_23,X_INTRODUCED_25,X_INTRODUCED_27,X_INTRODUCED_29,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_46,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_56,X_INTRODUCED_58];
array [1..31] of var int: X_INTRODUCED_132 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_3,X_INTRODUCED_6,X_INTRODUCED_9,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_25,X_INTRODUCED_27,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_34,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_45,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..61] of var int: X_INTRODUCED_136 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,objective];
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_11],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_48],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_52],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_4],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_9],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_27],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_3],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_4],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_7],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_21],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_50],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_51],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_5],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_6],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_18],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_8],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_16],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_35],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_15],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_23],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_13],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_14],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_36],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_10],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_20],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_22],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_17],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_35],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_12],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_19],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_47],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_26],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_58],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_59],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_45],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_31],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_44],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_49],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_42],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_23],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_28],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_38],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_48],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_56],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_24],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_32],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_54],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_32],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_23],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_28],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_38],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_43],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_48],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_33],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_37],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_40],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_25],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_33],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_34],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_38],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_39],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_26,X_INTRODUCED_28],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_26,X_INTRODUCED_33],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_29],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_30],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_40],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_30],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_44],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_42],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_49],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_31,X_INTRODUCED_52],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_36],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_42],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_36],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_35,X_INTRODUCED_39],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_35,X_INTRODUCED_44],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_37],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_43],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_37,X_INTRODUCED_46],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_49],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_39,X_INTRODUCED_43],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_40,X_INTRODUCED_41],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_41,X_INTRODUCED_47],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_42,X_INTRODUCED_53],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_50],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_55],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_44,X_INTRODUCED_45],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_45,X_INTRODUCED_47],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_46,X_INTRODUCED_52],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_46,X_INTRODUCED_56],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_54],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_55],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_56],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_48,X_INTRODUCED_50],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_48,X_INTRODUCED_51],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_49,X_INTRODUCED_55],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_50,X_INTRODUCED_53],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_50,X_INTRODUCED_58],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_51,X_INTRODUCED_54],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_51,X_INTRODUCED_58],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_52,X_INTRODUCED_53],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_53,X_INTRODUCED_59],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_54,X_INTRODUCED_57],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_55,X_INTRODUCED_57],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_56,X_INTRODUCED_57],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_56,X_INTRODUCED_59],-4);
constraint array_bool_or([X_INTRODUCED_119,X_INTRODUCED_120],true);
constraint jacop_cumulative(X_INTRODUCED_123,X_INTRODUCED_122,X_INTRODUCED_121,22);
constraint jacop_cumulative(X_INTRODUCED_126,X_INTRODUCED_125,X_INTRODUCED_124,22);
constraint jacop_cumulative(X_INTRODUCED_129,X_INTRODUCED_128,X_INTRODUCED_127,37);
constraint jacop_cumulative(X_INTRODUCED_132,X_INTRODUCED_131,X_INTRODUCED_130,19);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_57,objective],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_58,objective],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_59,objective],-2);
constraint int_lin_le_reif(X_INTRODUCED_62,[X_INTRODUCED_42,X_INTRODUCED_36],-10,X_INTRODUCED_119):: defines_var(X_INTRODUCED_119);
constraint int_lin_le_reif(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_42],-4,X_INTRODUCED_120):: defines_var(X_INTRODUCED_120);
solve :: int_search(X_INTRODUCED_136,smallest,indomain_min,complete) minimize objective;
