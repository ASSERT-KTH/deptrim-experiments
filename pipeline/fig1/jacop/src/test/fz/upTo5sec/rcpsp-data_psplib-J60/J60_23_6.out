objective = 81;
s = array1d(1..60, [0, 0, 0, 5, 5, 10, 10, 7, 13, 2, 14, 8, 24, 11, 11, 18, 13, 3, 14, 24, 14, 8, 17, 21, 31, 41, 17, 2, 25, 9, 31, 13, 31, 41, 48, 13, 52, 56, 48, 48, 31, 41, 56, 56, 19, 34, 64, 44, 21, 55, 62, 31, 69, 75, 48, 62, 69, 73, 69, 79]);
----------
==========
